system/bluetooth_monitor/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.5)
+project(bluetooth_monitor)
+
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  set(CMAKE_CXX_EXTENSIONS OFF)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+### Dependencies
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
+
+### Target executable
+ament_auto_add_executable(bluetooth_monitor
+  src/bluetooth_monitor_node.cpp
```

Comment:
I think it is better to use `register_node_macro` instead of creating a redundant main function.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/862/commits/442ef82559aba18b5b402dff982a2eac803fc314